datadog_checks_dev/datadog_checks/dev/tooling/commands/validate/licenses.py

```diff
@@ -112,6 +159,40 @@
 
 EXTRA_LICENSES = {'BSD-2-Clause'}
 
+
+EXPLICIT_CP_ATTR_PATHS = {
+    'adodbapi': 'https://raw.githubusercontent.com/mhammond/pywin32/main/adodbapi/__init__.py',
+    'aerospike': 'https://raw.githubusercontent.com/aerospike/aerospike-client-python/master/setup.py',
+    'beautifulsoup4': 'https://bazaar.launchpad.net/~leonardr/beautifulsoup/bs4/download/head:/LICENSE',
+    'futures': 'https://raw.githubusercontent.com/agronholm/pythonfutures/master/concurrent/futures/__init__.py'
+}
```

Comment:
To reviewer: I'm not so sure about having a mapping of url paths to non-license files. This seems hacky to me and not easy to maintain.

Reply:
Removed